wafer/settings.py

```diff
@@ -259,6 +259,13 @@
 
 # Set this to False to disable registration
 WAFER_REGISTRATION_OPEN = True
+# Can be 'ticket' for Quicket tickets or 'form' for a classic form
+WAFER_REGISTRATION_MODE = 'ticket'
+
+# For REGISTRATION_MODE == 'form': The form to present, and the Group to
+# associate the responses with in the KV store
+WAFER_REGISTRATION_FORM = 'wafer.users.forms.ExampleRegistrationForm'
+WAFER_REGISTRATION_GROUP = 'Registration'
```

Comment:
We currently have management/commands/wafer_add_default_groups.py to easily add the page edit and talk review groups. The "Registration" group should arguably be added to that list, so it's in place out of the box.